### PR TITLE
add traffic light user and admin pages

### PR DESCRIPTION
**Pull request by Ths2-9Y-LqJt6**

### CODE REVIEW

**Comment by Ths2-9Y-LqJt6** [comment]
@krux702  - there's a lot of place holder text here for info/pics/text/whatever that you have  (see admin page, [starting here](https://github.com/synshop/rtfm/pull/32/files#diff-23b637fee53a6281f5dd8f572ad491b7R15)).  Should I leave as is and you'll fill in later, you want to give stuff to me and I'll put it in this PR?  I'm open!

File: docs/admins/Traffic-Light.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+
+For general end user info, see [users guide](/users/Projects/Traffic-Light/). Otherwise, for nitty gritty technical info, read on!
+
+## Acquiring
+
+Where did we get this? How do we get another if ours explodes? Exercise left to reader.
+
+## Lights
+
+Lights are regular 100w bulbs.  You can swing the front of the fixture open to replace them when they go out.
```
Comment (Ths2-9Y-LqJt6) [inline docs/admins/Traffic-Light.md]: @krux702  - is this right? I did want to publish it upfront and not wait 'til later.

Comment (krux702) [reply docs/admins/Traffic-Light.md]: They are LED bulbs, not 100w

Comment (Ths2-9Y-LqJt6) [reply docs/admins/Traffic-Light.md]: Cool - can you recommend the text to put in here then? Maybe this?
> Lights are A19 LED 14w (100w equivalent)?

**Pull request merged**